test/e2e/manifest_test.go

```diff
@@ -549,6 +549,17 @@ RUN touch /file
 		Expect(output).To(ContainSubstring("Storing list signatures"))
 	})
 
+	It("push must retry", func() {
+		session := podmanTest.Podman([]string{"manifest", "create", "foo", imageList})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
+
+		push := podmanTest.Podman([]string{"manifest", "push", "--all", "--tls-verify=false", "--remove-signatures", "foo", "localhost:7000/bogus"})
+		push.WaitWithDefaultTimeout()
+		Expect(push).Should(Exit(1))
+		Expect(push.ErrorToString()).To(ContainSubstring("Failed, retrying in 1s ... (1/3)."))
```

Comment:
Yes, that looks good to me.  Any chance we can avoid the 3 seconds delay in the test?

Reply:
LGTM also. Strictly speaking, that doesn't actually confirm that there was a retry, so something like this would be more confidence-inspiring:
```go
  Expect(push.ErrorToString()).To(MatchRegexp("Copying blob.*Failed, retrying in 1s \\.\\.\\. \\(1/3\\).*Copying blob.*Failed, retrying in 2s"))
```
...but this is way in the weeds. I don't see a realistic way for "Failed, retrying" to appear but the retry itself not to happen.

Thank you for remembering to add a test!

Reply:
@edsantiago thanks I'll amend, could you let me know why current warning catch worked in some tests and failed in others ?

Reply:
ah warnings are not relayed in remote setup. @edsantiago thanks for pointing amended the test now.